File: 1768/solution.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        l1 = len(word1)
        l2 = len(word2)
        soln = []
        for i in range(min(l1, l2)):
            soln.extend([word1[i], word2[i]])
        if l1 > l2:
            soln.extend(word1[l2:])
        else:
            soln.extend(word2[l1:])
        return "".join(soln)

File: 1768/Solution.java
class Solution {
    public String mergeAlternately(String word1, String word2) {
        int l1 = word1.length();
        int l2 = word2.length();
        int min = Math.min(l1, l2);
        StringBuilder soln = new StringBuilder();
        for (int i = 0; i < Math.min(l1, l2); i++){
            soln.append(word1.charAt(i));
            soln.append(word2.charAt(i));
        }
        if (l1 > l2){
            soln.append(word1.substring(l2));
        } else {
            soln.append(word2.substring(l1));
        }
        return soln.toString();
    }
}